docs/Release_Guide/index.rst

```diff
@@ -82,6 +82,22 @@ software team and the respective funding institutions. For further inquiries or
 to report any bugs, please contact our dedicated support team in the
 `METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_.
 
+
+.. _releaseDependencies:
+
+********************
+Release Dependencies
+********************
+
+Software dependencies exist for the METplus components. The METviewer and
+METplus wrappers components have dependencies on other METplus components.
+
+Before METviewer is released, METdataio, METcalcpy, and METplotpy need
+to have been released.
+
+Before METplus wrappers is released, MET, METdataio, METcalcpy, METplotpy, and 
+METviewer all need to have been released.
```

Comment:
Is there a required order for METdataio/METcalcpy/METplotpy? I thought there are some inter-dependencies between those packages.

Reply:
@georgemccabe You are correct!  Thanks for paying close attention and catching my mistake.  I'll fix this up and request a re-review soon.